src/Phinx/Console/Command/AbstractCommand.php

```diff
@@ -113,7 +113,7 @@ public function bootstrap(InputInterface $input, OutputInterface $output)
         $this->loadManager($input, $output);
 
         if ($bootstrap = $this->getConfig()->getBootstrapFile()) {
-            $output->writeln('<info>using bootstrap</info> .' . str_replace(getcwd(), '', realpath($bootstrap)) . ' ');
```

Comment:
I wonder if we should move the realpath() part also into that new method.

Reply:
I'm not against it. If it were used everywhere `relativePath` is, I'd have included it.

Reply:
Sure, go ahead.